### ISSUE TITLE
Spark with Recessive Augment can send mana to only 6 mana pools with sparks

### ISSUE THREAD

**Issue by PanSzelescik** [Vazkii/Botania#2649]
# Version Information
Forge version:  2705 for Minecraft 1.12.2
Botania version:  r1.10-354
Tested on Sponge server! I don't know version

# Further Information
Steps to reproduce:
1. Place a Mana Pool with Spark and recessive augment
2. Płace near some mana pools with sparks (without augments)
3. Mana is sending to only 6 pools, others are empty
What I expected to happen:
Mana in all mana pools
What happened instead:
Mana is only in 6 mana pools

**Comment by williewillus** [comment]
how full is your source pool?

**Comment by Hazmatik** [comment]
I have seemed to have the same issue on pretty much every pack I have played with Botania for MC version 1.12.2 both on Sponge servers and on non-sponge servers, the only thing I've noticed is it's not always only 6 pools but it is certainly always not filling all the pools and when I click each spark with the wand of the forest it shows that they are all connected to the one with the recessive spark. It also seems that when the server restarts it will start sending mana to different pools but still not all the pools connected. Eventually all the pools that it sends mana to fills up and so does the pool the recessive spark is on, but the pools with the connected sparks that weren't filling never receive any mana.

Here is some screenshots of my setup on FTB Revelation: https://imgur.com/a/iDzWuNp

MC Version: 1.12.2
Forge Version: 14.23.4.2759
Botania Version: r1.10-354
